libs/ledger-live-common/src/load/speculos.ts

```diff
@@ -316,8 +316,11 @@ export function appCandidatesMatches(
           hackBadSemver(appCandidate.firmware),
           searchFirmware
         ))) &&
-    (!search.appVersion ||
-      semver.satisfies(appCandidate.appVersion, search.appVersion))
+    ((!search.appVersion &&
+      !appCandidate.appVersion.includes("prerelease") &&
+      !appCandidate.appVersion.includes("rc")) ||
+      (search.appVersion &&
+        semver.satisfies(appCandidate.appVersion, search.appVersion)))
```

Comment:
Actually I did some similar work for the pre-release tags on the manager `mustUpgrade` and `shouldUpgrade` methods [here](https://github.com/LedgerHQ/ledger-live/pull/332/files)
So if you goalr is just to support pre-release tags, you can use:

`semver.satisfies(appCandidate.appVersion, search.appVersion, { includePrerelease: true })`

Reply:
no, the goal is not to support prerelease. the goal is to NOT match a prerelease when `search.appVersion ` is false.

the fix of this PR isn't related to the semver, that code was simply moved.

Reply:
Oups ok my bad then 🙏